timemachine/fe/single_topology.py

```diff
@@ -178,23 +196,31 @@ def setup_dummy_interactions(
         if all([a in dga for a in idxs]):
             dummy_improper_idxs.append(tuple([int(x) for x in idxs]))
             dummy_improper_params.append(params)
+
+    # certain configuration of chiral states are symmetrizable
+    #
```

Comment:
will document, but . means dummy bond, and the right most diagram is disallowed because this function sets up dummy interactions, and a moeity with only core atoms is not a dummy interaction (by definition).